modules/Administration/PasswordManager.php

```diff
@@ -78,7 +78,8 @@ function clearPasswordSettings()
 $valid_public_key = true;
 if (!empty($_POST['saveConfig'])) {
     if ($_POST['captcha_on'] == '1') {
-        $handle = @fopen("http://www.google.com/recaptcha/api/challenge?k=" . $_POST['captcha_public_key'] . "&cachestop=35235354", "r");
+        if ((isset($_SERVER['HTTPS']) && 'on' === $_SERVER['HTTPS']) || (isset($_SERVER['SERVER_PORT']) && 443 === $_SERVER['SERVER_PORT']) ) { $protocol = 'https'; } else { $protocol = 'http'; }
+        $handle = @fopen("$protocol://www.google.com/recaptcha/api/challenge?k=".$_POST['captcha_public_key']."&cachestop=35235354", "r");
```

Comment:
Codacy/PR Quality Review - $_POST['captcha_public_key'] needs to be put into its own variable. Also it might be a good idea to urlencode it for security purposes.